src/models/user.js

```diff
@@ -153,6 +153,12 @@ export default (sequelize, DataTypes) => {
       foreignKey: 'userId',
       as: 'report'
     });
+    Users.hasMany(models.Highlights, {
+      foreignKey: 'userId',
+    });
+    Users.hasMany(models.HighlightComments, {
+      foreignKey: 'userId',
+    });
     Users.hasMany(models.Notification, {
```

Comment:
Good job @jideajayi11, but If there is no need for an alias when defining associations between models, then it is likely that there is no need for an association between the two models. Try to re-check how you implemented your models.

Reply:
it's fixed, thanks.